docs/rules/constructor-super.md

```diff
@@ -10,7 +10,9 @@ This rule checks whether or not there is a valid `super()` call.
 
 This rule is aimed to flag invalid/missing `super()` calls.
 
-The following patterns are considered problems:
+## Examples
```

Comment:
Regarding `arrow-parens` stuff about **If Statements** - yeah, I saw that and want to review that section of the doc separately - should I create a ticket to track that? Main focus was on the example headings and captions consistency.

Regarding the newly added h2 Examples, I'd really rather keep them - even for rules with no options - as it clearly differentiates the non-example content from the example content. If the concern is over consistency, I'm willing to go through all other rule files to add the h2 Examples if missing. I feel strongly that a consistent h2 Example heading is beneficial to the docs. Furthermore, if we add quick links at some point, having consistent headings will afford us the ability to initially implement that as a static list of links (which can later be updated to be dynamic as time permits). What do you think?